test/facets/Token/behaviors/enableSupportedChains.t.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity 0.8.19;
+
+import { TokenBridge } from "../TokenBridge.t.sol";
+import { ArbForkTest } from "../../../ArbForkTest.t.sol";
+import { ITokenBridge } from "../../../../contracts/facets/Token/ITokenBridge.sol";
+
+/// @title EnableSupportedChains
+/// @notice This contract tests the functionalities of the enableSupportedChains function
+contract EnableSupportedChains is ArbForkTest, TokenBridge {
+    error Ownable__NotOwner();
+    error TokenBridge__InvalidChain();
+    error TokenBridge__InvalidAddress();
+
+    event SupportedChainsEnabled(
+        string indexed destinationChain,
+        string indexed destinationAddress
+    );
+
+    /// @dev an example of a supported chain
+    string public supportedChain = "ethereum";
+    /// @dev an example of a destination address, it is a random address
+    string public destinationAddress =
+        "0x6513Aedb4D1593BA12e50644401D976aebDc90d8";
+    address public OWNER = makeAddr("Owner");
+    address public ALICE = makeAddr("Alice");
+
+    /// @dev address of the token proxy contract
+    address public tokenAddress;
+
+    function setUp() public virtual override {
```

Comment:
Add test.

Reply:
What kind of test should I add?

Reply:
It should test that the setup is done correctly:

```
function test_setUp() public override { 
     .... some reasonable logic to ensure the setup is as intended
}
```

This should be done in every test, even though it has not been done in the repo - this is a bad practice.

Since we'd have to do it everywhere, leave it even though we know its a good practice.

Reply:
Most of the setups didn't do anything besides deploying the contracts passing the gas service and the gateway address therefore this is the only thing I asserted here(dbb265668177497f52a6bd42f3523cb34ab6b79f)